setup.py

```diff
@@ -10,21 +10,14 @@
         ]
     },
     install_requires=[
-        'flask==0.12.4',
-        'jsonschema==2.5.1',
-        'sqlalchemy==1.0.8',
-        'sqlalchemy-utils>=0.32.21',
-        'psycopg2>=2.7',
-        'future>=0.16.0,<1.0.0',
+        'flask',
```

Comment:
I think it'd still be good to specify an expected range for these, even if it's pretty broad.

We should also pin SQLAlchemy to 1.3.3 in `requirements.txt`, since I think that's what we're targeting on the psqlgraph side.

Reply:
You got it 👍